src/main/kotlin/org/rust/ide/fixes/ConvertCfgNotMultipleArgsToMultipleNotsFix.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsMetaItem
+import org.rust.lang.core.psi.RsPsiFactory
+
+class ConvertMalformedCfgNotPatternToCfgAllPatternFix(element: RsMetaItem) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    private val fixText = run {
+        val metaItemArgsList = element.metaItemArgs?.metaItemList ?: return@run ""
+        if (metaItemArgsList.size < 2) {
+            return@run ""
+        }
+
+        val negatedArguments = metaItemArgsList.joinToString(", ") { "not(${it.text})" }
+
+        "Convert to `all($negatedArguments)`"
+    }
+
+    override fun getFamilyName(): String = "Fix `not` cfg-pattern"
+    override fun getText(): String = fixText
+
+    override fun isAvailable(project: Project, file: PsiFile, startElement: PsiElement, endElement: PsiElement): Boolean {
+        val metaItem = startElement as? RsMetaItem ?: return false
+        return metaItem.metaItemArgsList.size > 1
+    }
```

Comment:
We usually don't use `isAvailable` in quick fixes and assume that the fix is available if it was supplied by an annotator/inspection.

Reply:
The problem is that the fix doesn't work if there are zero arguments passed to `not()`. But it's still an error to do so
Would you suggest to instead do this check on the inspection level and not create the fix at all?

Reply:
We usually use `Fix.createIfCompatible` pattern. See for example `ChangeReprAttributeFix.createIfCompatible`